src/core/lookup.c

```diff
@@ -55,7 +55,10 @@ QuicLookupUninitialize(
         for (uint8_t i = 0; i < Lookup->PartitionCount; i++) {
             QUIC_PARTITIONED_HASHTABLE* Table = &Lookup->HASH.Tables[i];
             QUIC_DBG_ASSERT(Table->Table.NumEntries == 0);
+#pragma warning(push)
+#pragma warning(disable:6001)
```

Comment:
Instead of pragma ignores, I'd prefer DBG asserts. What is the error for?

Reply:
Uninitialized memory. So can't really assert it.

Reply:
It's assuming that Directory in the union in QUIC_HASHTABLE  is uninitialized.